frontend/src/views/auth/LoginView/LoginForm.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './LoginFrom.css';
+import validate from './LoginValidation';
+import LoginUseForm from './LoginUseForm';
+
+export const LoginForm = () => {
+  const { handleChange, handleSubmit, values, errors } = LoginUseForm(validate);
+
+  return (
+    <form className="form" onSubmit={handleSubmit}>
+      <label htmlFor="email">
+        Email
+        <input
+          name="email"
+          type="text"
```

Comment:
Trzymanie inputa w labelu jest semantycznie poprawne z tego co wiem, ale ogranicza troche mozliwosci jesli chodzi o stylowanie. Niekoniecznie do zmiany teraz. Wyjdzie dopiero to pewnie przy takim porzadniejszym stylowaniu :)